app/app.go

```diff
@@ -284,6 +290,7 @@ func NewOsmosisApp(
 		gammtypes.StoreKey, lockuptypes.StoreKey, claimtypes.StoreKey, incentivestypes.StoreKey,
 		epochstypes.StoreKey, poolincentivestypes.StoreKey, authzkeeper.StoreKey, txfeestypes.StoreKey,
 		bech32ibctypes.StoreKey,
+		tokenfactorytypes.StoreKey,
```

Comment:
We need to make a github issue to write the migration logic to initialize the store key
(Issue creation blocks merge, solving it does not)

Reply:
#689